### ISSUE TITLE
WCAG for Accessibility

### ISSUE THREAD

**Issue by abaporu-C** [abaporu-C/GROW-POS#47]
Tooltips are needed for accessibility. Labels should have descriptive information on a tooltip to show what is expected from a user.

Labels' text also need to be in a <strong></strong> tag for visual aid.

Anything else that you might think about, please implement.

**Comment by abaporu-C** [comment]
Closed with pull request #147
